src/main/java/com/xebialabs/restito/server/StubServer.java

```diff
@@ -36,6 +36,7 @@
     private final List<Call> calls = new CopyOnWriteArrayList<>();
     private final List<Stub> stubs = new CopyOnWriteArrayList<>();
     private final HttpServer simpleServer;
+    private boolean registerCalls = false;
```

Comment:
I think, it should be `true` by default. Otherwise, after upgrade, tests will stop working for all existing users, that rely on registered calls.